### PR TITLE
Feature/bc 307 transformation hpi schul cloud in d bildungscloud

### PR DESCRIPTION
**Pull request by monaweitzenberg**
# Description
theme and content transformation HPI Schul-Cloud into dBildungscloud

## Links to Tickets or other pull requests
https://ticketsystem.hpi-schul-cloud.org/browse/BC-307

## Changes
Platform

adjust primary and secondary color
exchange Logo
exchange default-file: instance_cfg.yml (line 3-6) title and contact
delete the PartnerTeam, About pages
clean up the footer
adjust imprint
Landing-Page

adjust primary and secondary color
exchange Logo
delete sections: partner; newsletter

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: theme/open/views/help/people.hbs [hunk 1]
```diff
@@ -19,7 +19,7 @@
 						</div>
 						<div id="collapse-0" class="collapse show in">
 							<div class="card-block">
-								Für die Meldung von Problemen oder Wünschen hinsichtlich der Bedienung der HPI Schul-Cloud verwende bitte das Kontaktformular im Hilfebereich oder sende eine E-Mail an <br><a href="mailto:feedback@hpi-schul-cloud.de" target="blank" rel="noopener" class="externalLink">feedback@hpi-schul-cloud.de</a>
+								Für die Meldung von Problemen oder Wünschen hinsichtlich der Bedienung der HPI Schul-Cloud verwende bitte das Kontaktformular im Hilfebereich oder sende eine E-Mail an <br><a href="mailto:dataportlernen.cloud@dataport.de" target="blank" rel="noopener" class="externalLink">dataportlernen.cloud@dataport.de</a>
```
Comment (pkleybolte) [inline theme/open/views/help/people.hbs:22]: Better with translation

File: theme/thr/views/help/people.hbs [hunk 2]
```diff
@@ -33,7 +33,7 @@
 						</div>
 						<div id="collapse-2" class="collapse show in">
 							<div class="card-block">
-								Für die Meldung von Problemen oder Wünschen hinsichtlich der Bedienung der Thüringer Schul-Cloud verwende bitte das Kontaktformular im Hilfebereich oder sende eine E-Mail an <br><a href="mailto:feedback@hpi-schul-cloud.de" target="blank" rel="noopener" class="externalLink">feedback@hpi-schul-cloud.de</a>
+								Für die Meldung von Problemen oder Wünschen hinsichtlich der Bedienung der Thüringer Schul-Cloud verwende bitte das Kontaktformular im Hilfebereich oder sende eine E-Mail an <br><a href="mailto:dataportlernen.cloud@dataport.de" target="blank" rel="noopener" class="externalLink">dataportlernen.cloud@dataport.de</a>
```
Comment (pkleybolte) [inline theme/thr/views/help/people.hbs:36]: Missing Translation

**Review by pkleybolte — APPROVED** [review]

File: theme/open/views/help/people.hbs (hunk 1 above)
Comment (monaweitzenberg) [reply theme/open/views/help/people.hbs:22]: Translation is not part of the ticket. In addition, it appears that the site is no longer live.

File: theme/thr/views/help/people.hbs (hunk 2 above)
Comment (monaweitzenberg) [reply theme/thr/views/help/people.hbs:36]: Translation is not part of the ticket. In addition, it appears that the site is no longer live.

**Review by pkleybolte — APPROVED** [review]

**Pull request merged**